Automattic/wp-calypso
# Update: Remove subscribe actions for unverfied email users

## Proposed Changes

* This PR updates the buttons that let users subscribe to sites but send them an error message asking them to verify the emails first. 


See

https://github.com/Automattic/wp-calypso/assets/115071/ed90d0d1-f1b5-45fe-909a-1f929b85e404

Screenshot for translators. 
<img width="790" alt="Screenshot 2024-06-12 at 5 09 13 PM" src="https://github.com/Automattic/wp-calypso/assets/115071/1e98a16f-a6c3-40c3-a563-91bda643d7e3">



## Why are these changes being made?
The changes are that we check the subscribe buttons and other buttons and send a notice instead. 

*

## Testing Instructions
Create a new unverfied email user. 

Go to /read
Notice that you can't subscribe to a site or follow a conversation. 

Go to /read/subscriptions
Notice that you can't follow or subscribe to a site. 
The UI elements are not even there. 


Use a regual user
Notice that when you go to /read and /read/subscritions the UI elements are there so you can follow and subscribe to the site. 

## Pre-merge Checklist
